### PR TITLE
Init vendir

### PR DESCRIPTION
**Pull request by QuentinBisson**
This PR:

- inits keda-app with vendir
- creates cilium network policy

Be aware that we can only use a tag of KEDA right now because of kubernetes compatibily reasons.
- v2.11 works starting from Kubernetes 1.25
- v2.12 works starting from Kubernetes 1.26

TODOs:
- [x] Deploy in one of our clusters (tested in a 19.2.0 cluster)
    - [x] Add cilium network policies (upstream PR https://github.com/kedacore/charts/pull/558)
    - [x] Ensure no PSS violation and no PSP usage   
- [x] Retag image
    - [x] Added support for our registry override (upstream PR https://github.com/kedacore/charts/pull/557 in latest version (not applicable to us))
- [x] Test basic triggers and HPA to be sure the app works
- [x] Rewrite all documentations (READMEs)
- [ ] Basic alerting and [dashboards](https://github.com/giantswarm/dashboards/pull/398)

### Testing

Description on how keda can be tested.

- [x] fresh install works
  - [x] AWS

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

**Comment by QuentinBisson** [comment]
App was tested on a workload cluster with a trigger based on REDIS

File: README.md [hunk 1]
```diff
@@ -1,70 +1,61 @@
...
+- [Using our web interface](https://docs.giantswarm.io/ui-api/web/app-platform/#installing-an-app).
+- By creating an [App resource](https://docs.giantswarm.io/ui-api/management-api/crd/apps.application.giantswarm.io/) in the management cluster as explained in [Getting started with App Platform](https://docs.giantswarm.io/app-platform/getting-started/).
 
-If you have access to the Kubernetes API on the management cluster, you could create
-the App CR and ConfigMap directly.
+## Upgrading keda version
 
-Here is an example that would install the app to
-workload cluster `abc12`:
+The contents of the `helm` folder are being generated by the `make` target called `make update-chart`.
```
Comment (QuantumEnigmaa) [inline README.md]:
```suggestion
The content of the `helm` folder are being generated by the `make` target called `make update-chart`.
```
I think `content` is invariable

**Review by QuantumEnigmaa — COMMENTED** [review]
Just out of my own curiosity : why using vendir for this particular app instead of chart dependency ?

**Comment by QuentinBisson** [comment]
@QuantumEnigmaa Because this app was supposed to be maintained by @giantswarm/team-turtles and they prefer to use vendir

File: README.md (hunk 1 above)
Comment (QuentinBisson) [reply README.md]: fixed

**Review by QuantumEnigmaa — APPROVED** [review]
LGTM

**Pull request merged**